src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.nav_menu {
+  display: flex;
+  justify-content: flex-start;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_menu a {
+  margin: 0 0 0 20px;
+  color: #000;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 60px;
+  text-align: center;
+}
+
+.nav_menu a:hover,
+.nav_menu a[data-qa='hover']:hover {
+  color: var(--blue-color);
+}
+
+.nav_menu li {
   margin: 0;
 }
```

Comment:
Avoid using tag names for styling. It's better to style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. In this case, you can remove `margin: 0;` from `.nav_menu li` because it's already set in the universal selector (*).